File: Computing/MTR/O_2023/ANIMAL_2023_Nikola_406_20.py
p1_animals = []

while True:
    p1_animal = input("Player 1, please enter an animal: ")
    p1_animal = p1_animal.lower()
    p1_animals.append(p1_animal)

    more = input("Do you still want to enter an animal? [Y/N]")
    if more.upper() == "Y":
        continue
    elif more.upper() == "N":
        break

p2_guess = input("Player 2, please enter your guess: ")
p2_guess = p2_guess.lower()

File: Computing/C_Practice/practice_papers/O_2023/ANIMAL_2023_Nikola_406_20.c
#include <stdio.h>
#include <ctype.h>

int main() {
    int num_of_animals = 5;
    char *p1_animals;
    char *p2_guess;

    for (int i = 0; i < num_of_animals; i++) {
        printf("Player 1, please enter an animal: ");
        scanf("%s", p1_animals);

        for (int i = 0; p1_animals[i]; i++) {
            p1_animals[i] = tolower(p1_animals[i]);
        }
        
        printf("Player 2, please enter a guess: ");
        scanf("%s", p2_guess);

        for (int i = 0; p2_guess[i]; i++) {
            p2_guess[i] = tolower(p2_guess[i]);
        }
        
    }
}
